src/pages/[id].page.jsx

```diff
@@ -44,6 +45,7 @@ export default function Details({ componentName }) {
     <Layout pageTitle={componentName}>
       <div className="details-page">
         <DetailsBanner name={componentName} image={image} />
+        <h1>{componentName}</h1>
```

Comment:
There are two h1 elements on this page, we should remove this one on line 48.

Reply:
Thanks, good catch!